Automattic/wp-calypso
# Design refresh for thank you page upsell

Fixes https://github.com/Automattic/dotcom-forge/issues/4484

## Proposed Changes

* Updated design of `ThankYouUpsell` per VLoPHWqcewxNKZYjjkDu3O-fi-2996%3A3281. The email upsell is the only version implemented yet.
* Updated the href of the email upsell CTA so that it leads to the post-checkout upsell page.

| Before | After |
| - | - |
| ![desktop-before](https://github.com/Automattic/wp-calypso/assets/1101677/405aa0ee-6679-41f8-8bb4-221e114e5648) | ![desktop-after](https://github.com/Automattic/wp-calypso/assets/1101677/90e4cb47-e4a3-4665-8d2e-1a26b7042812) |
| ![mobile-before](https://github.com/Automattic/wp-calypso/assets/1101677/090650e4-9114-4329-9aae-92900a8dd744) | ![mobile-after](https://github.com/Automattic/wp-calypso/assets/1101677/acc4fe2e-c30d-4ce1-8859-22d2e388b6fb) |

## Testing Instructions



- If you don't have a domain previously purchased, purchase one.
- If you have a domain handy, get the respective receipt ID using the following steps:
  - Go to Upgrades > Purchases > Billing History
  - Look for the domain purchase and click on "View receipt" and you should be able to find the receipt ID
- Visit `/checkout/thank-you/:site/:receipt_id`
- Assert that the page matches the design mocks on desktop and on mobile

## Pre-merge Checklist
